Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -36,6 +34,7 @@ class XcodeSpec: QuickSpec {
 		}
 
 		describe("determingSwiftInformation") {
+			let currentSwiftVersion = swiftVersion.single()?.value
```

Comment:
This expectation:

```swift
expect(currentSwiftVersion?.isEmpty) == false
```

should fail if `currentSwiftVersion` is `nil`. I thought it might be better to have that test fail as an indicator that things aren't working as we would expect rather than having the tests crash. What  do you think @mdiep?